apps/server/src/modules/lesson/repository/lesson.repo.ts

```diff
@@ -44,7 +45,7 @@ export class LessonRepo extends BaseRepo<LessonEntity> {
 				$match: {
 					contents: {
 						$elemMatch: {
-							user: userId,
```

Comment:
Interessting ...do it work before without this change?

Reply:
it didn'tt work before, that's why we changed it